keyboards/eyeohdesigns/Readme.md

```diff
@@ -0,0 +1,13 @@
+Eye Oh Designs
```

Comment:
Could this be renamed to it's all lower case?

Reply:
Requested changes made.